docs/semgrep-appsec-platform/dashboard-beta.md

```diff
@@ -0,0 +1,121 @@
+<dd>A graph showing the time to remediate over the given time period.</dd>
+</dl>
+
+## Filters and configuration
+
+Use the filters to gain a top-level view or zoom in to a specific period of time. Create quarterly overviews or recent incident statements for various AppSec stakeholders by specifying the following filters:
+
+- Time period
+- Project (repository)
+- Semgrep product or type of scan (SAST, SCA, or Secrets)
+
+Configurations set here apply to the following panes:
+
+- Production backlog
+- Secure guardrails
+
+### Recommended priority filters
+
+By default, **<i class="fa-solid fa-toggle-large-on"></i> Recommended priority** filters are enabled. These filters display any finding that is **Critical** or **High** severity in **addition** to being:
+
+- High confidence - if the finding is from Semgrep Code.
+- Reachable - if the finding is from Semgrep Supply Chain.
+- Valid - if the finding is from Secrets.
+
+tk add links to definitions
+
+## Production backlog
+
+This pane displays analytics related to findings detected in your primary or default branch. This typically means that the finding, usually a security issue, has made it to production environments.
+
+### Key metrics 
+
+| Key metrics    | Description |
+| -------------- | ------ |
+| Total new      | |
+| Total fixed    | |
+| Total ignored  | |
+| Total net new  | |
+
+### Charts
+
+| Chart | Description |
+| -------  | ------ |
+| Open backlog         | Open findings over the chosen time period. This tracks the total findings from each scan and displays them. Lower values are better. |
+| Backlog activity | Displays the number of new, net new, fixed, and ignored findings. A greater **Fixed** value is better. |
+
+## Secure guardrails
+
+This provides an overview of how secure guardrails in **PR or MR comments** are used in your organization. Other guardrail interfaces, such as the IDE or `pre-commit`, are not counted in this section.
+
+![Secure guardrails pane](/img/dashboard-guardrails.png)
+_**Figure**. Secure guardrails pane._
+
+### Key metrics
+
+| Key metrics    | Description |
+| -------------- | ------ |
+| Findings shown to devs      | Percentage of findings shown to devs in PR or MR comments against the total findings count. An upward or stable trend is better. |
+| Findings fixed before backlog      | Displays the percentage of findings that were fixed before they could be detected in a default branch or production backlog against the total findings count. An upward or stable trend is better. |
+
+### Charts
+
+| Chart | Description |
+| -------  | ------ |
+| Secure guardrails adoption  | Percent of new findings shown to developers over the specified time period. An upward or stable trend is better. |
+| Guardrails activity | This chart displays a breakdown of the status of findings shown to developers; whether they were ignored, fixed, or remained open. A greater **Fixed** value is better. |
+
+## Most findings by project
```

Comment:
@milanwilliams Why do we call it "Most findings by project" rather than "Findings by project"?

Reply:
we want to surface the projects with the _most_ findings so users can figure out where to pay attention / allocate resources, so we labeled it "most" to emphasize the ordering is most -> least (rather than just an alphabetical list of projects or something)

Reply:
i wonder if we should add something in about this surfacing "hotspots" - right now, we say that lower values are better, but i think we want to focus on the fact that they should pay attention and prioritize the projects with the most findings cc @milanwilliams

Reply:
Added the ff callout:
![image](https://github.com/user-attachments/assets/20e5bd91-328e-4788-bda7-56475e036b1f)